content/pages/quickstart.md

```diff
@@ -25,7 +25,8 @@ This guide will lead you through creating a user site at `username.github.io`.
 {% data reusables.repositories.create_new %}
 1. Enter `username.github.io` as the repository name. Replace `username` with your {% data variables.product.prodname_dotcom %} username. For example, if your username is `octocat`, the repository name should be `octocat.github.io`.
    ![Repository name field](/assets/images/help/pages/create-repository-name-pages.png)
-{% data reusables.repositories.sidebar-settings %}
+2. Under your repository name, click **{% octicon "gear" aria-label="The gear icon" %} Settings**.
```

Comment:
I think it would be worth updating this image across the whole docs site, so let's re-add the content of lines 28 and 29 back into the `{% data reusables.repositories.sidebar-settings %}` reusable.